cdisutils/dictionary.py

```diff
@@ -14,7 +14,7 @@ def sort_dict(tree, remove_keys=None):
     if isinstance(tree, dict):
         return {
             key: sort_dict(tree[key], remove_keys=remove_keys)
-            for key in tree.keys() if key not in remove_keys
+            for key in list(tree.keys()) if key not in remove_keys
```

Comment:
These `list()`s get added by conversion tools because the tools don't know if the Python 2 code expected `keys()` to return a list or not.

In most of these cases, it looks like we're just iterating through the keys, so it should be fine to skip the `list` and do something like
```suggestion
            for key in tree if key not in remove_keys
```